test-support/helpers/ember-tooltips.js

```diff
@@ -222,3 +222,20 @@ export function assertTooltipSpacing(assert, options) {
         - Tooltip should be on the ${side} side of the target: ${isSideCorrect}.
         - On the ${side} side of the target, the tooltip should be ${spacing}px from the target but it was ${actualSpacing}px`);
 }
+
+export function assertTooltipContent(assert, options = {}) {
+  const { contentString } = options;
+
+  if (contentString === undefined) {
+    Ember.assert('You must specify a contentString property in the options parameter');
+  }
+
+  const $tooltip = getTooltipFromBody(options.selector);
+  const tooltipContent = $tooltip.text().trim();
+
+  assert.equal(
```

Comment:
nit: not blocking, could you follow the syntax we did elsewhere in this file...

```
assert.equal(a, b, 'some description');
```

PS: I like that you added the tooltipContent and contentString in the description